5/profile

xx3u opened the pull request:
Issue #4 
This branch reflects all changes including updates on branch Authorization (PR) and Readme (merged PR) as compared to master.
- Update Profile model based on AbstractUser;
- Update pytest coverage;
- Add Readme for badges;
- Add forms using external installed requirements;
- Update features in templates;
- Add tests functional and unit

xx3u committed:
Add CustomUser based on AbstractUser

xx3u committed:
Update Profile model using AbstractUser;
Update templates for some features;
Add url on profile for detailed information;
Add form using external instal requirements;
Update tests;

khazidhea commented:
Я не понимаю. В чем здесь разница с #1 ?

Здесь изменений больше, но вижу много пересекающихся. Читать очень неудобно.

xx3u committed:
Fix merge conflicts

xx3u committed:
Update migrations

xx3u committed:
Fix test for redirect as follow=True

khazidhea commented on website/models.py:
```diff
@@ -2,15 +2,25 @@
+class Profile(AbstractUser):
     birthday = models.DateField(null=True, blank=True)
-    contact_number = models.CharField(max_length=20, blank=True)
+    contact_number = models.CharField(max_length=20, blank=True, null=True)
```
Убрать null=True

khazidhea commented on website/templates/base.html:
```diff
@@ -1,43 +1,46 @@
+    {% if title %}
+      <title>Superheros - {{ title }}</title>
+    {% else %}
+      <title>Superheros</title>
```
SuperheroEs

khazidhea commented on website/templates/base.html:
```diff
@@ -1,43 +1,46 @@
+    <meta name="description" content="">
+    <meta name="author" content="">
+    {% if title %}
+      <title>Superheros - {{ title }}</title>
```
SuperheroEs

khazidhea commented on website/test/functional/test_views.py:
```diff
@@ -58,3 +58,13 @@ def test_register_view(db, client, data):
+
+def test_profile(db, client, data):
+    response = client.get('/profile/', follow=True)
+    assert response.status_code == 200
```
Проверить что загрузился правильный профайл

xx3u committed:
- Update Profile models for additional details;

- Add Story model (like posts from superheroes);

- Delete unnecessary Image model;

xx3u committed:
Fix type in template

xx3u committed:
Update test for profile details view

xx3u committed:
Update background and view

xx3u committed:
Update tests for register views
